plugins/rtmp-services/data/services.json

```diff
@@ -842,5 +842,14 @@
                 "max audio bitrate": 160
             }
         }
```

Comment:
There's a missing comma here and instead it is at the bottom of the new section. This makes the file invalid JSON.